backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -141,6 +141,7 @@ def test_qnn_backend_element_wise_ceil(self):
 
     def test_qnn_backend_element_wise_div(self):
         eps = 1e-03
+        torch.manual_seed(8)
```

Comment:
Should just set a manual seed for all the test here?

Reply:
This test case is checking the functionality of the div op, and using the random input may have a higher possibility to fail than other test case. For other op, they often can pass with the random input.
From my view, I think we could only set this manual seed for this test case for now and check if other test also have this kind of issue.
But yes, if we would like to use the manual seed for all the test, please kindly let me know and will update it ASAP, thank you!